cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -27,6 +27,12 @@ extensions:
       configDir: "${SPLUNK_COLLECTD_DIR}"
   zpages:
     #endpoint: 0.0.0.0:55679
+  memory_ballast:
+    # In general, the ballast should be set to 1/3 of the collector's memory, the limit
+    # should be 90% of the collector's memory.
+    # The simplest way to specify the ballast size is set the value of SPLUNK_BALLAST_SIZE_MIB env variable.
+    # Alternatively, the --mem-ballast-size-mib command line flag can be passed and take priority.
```

Comment:
Delete this to avoid advertising the old approach?

Reply:
Done.